config/logging/CanadaPubSecALZ-main/logging.parameters.json

```diff
@@ -0,0 +1,149 @@
+{
```

Comment:
Appreciate you taking the time to walk through how this is applied.  

Are there any new `logging.parameters` that now need to be moved over into our [`cdssns-main` params](https://github.com/cds-snc/CanadaPubSecALZ/blob/main/config/logging/cdssnc-main/logging.parameters.json)?

Reply:
all new parameters are added in cdssnc-main